.coderabbit.yaml

```
@@ -0,0 +1,14 @@
+reviews:
+  profile: "chill"
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: true
+  review_status: true
+  collapse_walkthrough: false
+  auto_review:
+    enabled: true
+    drafts: true
+chat:
+  auto_reply: true
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)